cmd/pool/account.go

```diff
@@ -172,30 +199,42 @@ func newAccount(ctx *cli.Context) error {
 	return nil
 }
 
-func printAccountFees(client poolrpc.TraderClient, amt btcutil.Amount,
-	confTarget uint32) error {
+func printAccountFees(
+	client poolrpc.TraderClient,
+	amt btcutil.Amount,
+	satPerVByte uint64,
+	confTarget uint32,
+) error {
+	if confTarget != 0 {
+		resp, err := client.QuoteAccount(context.Background(),
+			&poolrpc.QuoteAccountRequest{
+				AccountValue: uint64(amt),
+				Fees: &poolrpc.QuoteAccountRequest_ConfTarget{
+					ConfTarget: confTarget,
+				},
+			})
+		if err != nil {
+			return fmt.Errorf("unable to estimate on-chain fees: "+
+				"%v", err)
+		}
 
-	req := &poolrpc.QuoteAccountRequest{
-		AccountValue: uint64(amt),
-		Fees: &poolrpc.QuoteAccountRequest_ConfTarget{
-			ConfTarget: confTarget,
-		},
-	}
-	resp, err := client.QuoteAccount(context.Background(), req)
-	if err != nil {
-		return fmt.Errorf("unable to estimate on-chain fees: "+
-			"%v", err)
-	}
+		feeRate := chainfee.SatPerKWeight(resp.MinerFeeRateSatPerKw)
+		newSatPerVByte := float64(feeRate.FeePerKVByte()) / 1000
+		fmt.Println("-- Account Funding Details --")
+		fmt.Printf("Amount: %v\n", amt)
+		fmt.Printf("Confirmation target: %v blocks\n", confTarget)
+		fmt.Printf("Fee rate (estimated): %.1f sat/vByte\n", newSatPerVByte)
+		fmt.Printf("Total miner fee (estimated): %v\n",
+			btcutil.Amount(resp.MinerFeeTotal))
 
-	feeRate := chainfee.SatPerKWeight(resp.MinerFeeRateSatPerKw)
-	satPerVByte := float64(feeRate.FeePerKVByte()) / 1000
+		return nil
+	}
 
 	fmt.Println("-- Account Funding Details --")
 	fmt.Printf("Amount: %v\n", amt)
-	fmt.Printf("Confirmation target: %v blocks\n", confTarget)
-	fmt.Printf("Fee rate (estimated): %.1f sat/vByte\n", satPerVByte)
-	fmt.Printf("Total miner fee (estimated): %v\n",
-		btcutil.Amount(resp.MinerFeeTotal))
+	// TODO(positiveblue): print Total miner fee (estimated) for the
```

Comment:
I don't think we can really say that in the case here. Because above (with the conf target) we ask the wallet for an estimation, so it knows how many inputs it's going to use and whether there is a change output. But here we don't know anything about the actual transaction that's going to be sent out other than the fee rate.